libretro/libretro-core.c

```diff
@@ -1396,19 +1412,40 @@ void retro_set_environment(retro_environment_t cb)
       {
          "vice_video_options_display",
          "Show Video Options",
-         "Core options page refresh required.",
+         "Should be used with RetroArch Video/Scaling Settings: Aspect Ratio: Core provided\nCore options page refresh required.",
          {
             { "disabled", NULL },
             { "enabled", NULL },
             { NULL, NULL },
          },
          "disabled"
       },
-#if !defined(__PET__) && !defined(__CBM2__)
+
+
+// VIDEO OPTIONS START - RetroArch Core Options Menu
+
+#if defined(__X64__) || defined(__X64SC__) || defined(__X128__) || defined(__VIC20__) || defined(__PLUS4__)
+      {
+         "vice_aspect_ratio",
+         "Aspect Ratio",
+         "Automatic applies Pixel Aspect correction for the current system model\nPAL/NTSC force specific Pixel Aspect & can be used to stretch/compress opposite model\n1:1 does no Pixel Aspect correction",
+         {
+            { "auto", "Automatic" },
+            { "pal", "PAL" },
+            { "ntsc", "NTSC" },
+            { "raw", "1:1" },
+            { NULL, NULL },
+         },
+         "auto"
+      },
+#endif
+
+#if !defined(__PET__) && !defined(__CBM2__)     // This should always be left ON, serious performance and graphics glitches when OFF
       {
          "vice_border",
          "Display Borders",
-         "All cores except 'x64sc' will reset when this option is changed.",
+         "Leave 'ON' otherwise may affect performance - Use Border Cropping options instead\n"

```

Comment:
Please revert this change. This does not affect performance. Borders off is a standard vice option, no need to tell people to leave it "ON"

Reply:
If you do the testing, you'll see that function is buggy. It absolutely reduces performance, including sprite glitching.

@sonninnos proposed eliminating that option. I only Clarified that it should always be left on because of its adverse effects on performance and it being ON a requirement to use any zoom/cropping.

Reply:
Ok I thought it was fine unless the old X64 was used (not X64sc)

Reply:
It doesn't matter which C64 core is used AFAIK.  Simple test, try Commando in ViceGTK or RA.

I prefer Tony's change for this however, which is to hard-set it ON and remove it from the menu.